src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/RemoteBuildConfiguration.java

```diff
@@ -1270,9 +1562,37 @@ public boolean configure(StaplerRequest req, JSONObject formData) throws FormExc
             return super.configure(req, formData);
         }
 
+        public FormValidation doCheckJob(
```

Comment:
Use `@Restricted(NoExternalUse.class)` for new form validation logic. Otherwise it formally becomes public API